app/v2/controllers/GuaranteeBalanceController.scala

```diff
@@ -67,19 +70,27 @@ class GuaranteeBalanceController @Inject() (
   def postRequest(grn: GuaranteeReferenceNumber): Action[JsValue] =
     authenticate().io(parse.json) {
       implicit request =>
-        (for {
-          _      <- validateAcceptHeader(grn)
-          parsed <- parseJson(request, grn)
-          auditInfo = AuditInfo(parsed, grn, request.internalId)
-          _                <- lockService.lock(grn, request.internalId).asPresentation(auditInfo, auditService)
-          _                <- validationService.validate(parsed).asPresentation(auditInfo, auditService)
-          internalResponse <- routerService.request(grn, parsed).asPresentation(auditInfo, auditService)
-          hateoas          <- EitherT.right[PresentationError](HateoasResponse(grn, internalResponse))
-          _ = auditService.balanceRequestSucceeded(auditInfo, internalResponse.balance)
-        } yield hateoas).fold(
-          presentationError => Status(presentationError.code.statusCode)(Json.toJson(presentationError)),
-          result => Ok(result)
-        )
+        if (appConfig.enablePhase5) {
+          (for {
+            _      <- validateAcceptHeader(grn)
+            parsed <- parseJson(request, grn)
+            auditInfo = AuditInfo(parsed, grn, request.internalId)
+            _                <- lockService.lock(grn, request.internalId).asPresentation(auditInfo, auditService)
+            _                <- validationService.validate(parsed).asPresentation(auditInfo, auditService)
+            internalResponse <- routerService.request(grn, parsed).asPresentation(auditInfo, auditService)
+            hateoas          <- EitherT.right[PresentationError](HateoasResponse(grn, internalResponse))
+            _ = auditService.balanceRequestSucceeded(auditInfo, internalResponse.balance)
+          } yield hateoas).fold(
+            presentationError => Status(presentationError.code.statusCode)(Json.toJson(presentationError)),
+            result => Ok(result)
+          )
+        } else {
+          // Let the client know Phase 5 is not yet available.
+          val presentationError = PresentationError.notAcceptableError(
+            "CTC Traders API version 2 is not yet available. Please continue to use version 1 to submit transit messages."
```

Comment:
CTC Guarantee Balance API, not Traders.

Reply:
Indeed - copied from the ticket :scream_cat:

Reply:
Will change.